### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lbusschaert**
- Added logo and changed title
- Spell out FAO
- Extra line on the testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# AquaCrop
+# FAO AquaCrop
+
+<img src="https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcSEmcLG0zbWXIaLClg09v77ZccbFH_zuDVRbH-eBLxAdmcZ4nZ7" align="right" width="200px">
 
 AquaCrop v7.0 is released as open-source Fortran code, 
 developed at KU Leuven and FAO (FAO and KU Leuven copyright),
```
Comment (KUL-RSDA) [inline README.md:6]: Could you change as part of this PR:
developed at KU Leuven and the Food and Agriculture Organization (FAO) of the United Nations (FAO and KU Leuven copyright),

**Review by KUL-RSDA — APPROVED** [review]
Great! For findability/visibility one could once introduce the abbreviation FAO, see my comment.

File: README.md (hunk 1 above)
Comment (lbusschaert) [reply README.md]: Done!

**Review by mbechtold — APPROVED** [review]
Great. By mistake, I approved as KUL-RSDA previously. Now with mbechtold and this is ready to merge.

**Comment by gdelannoy** [comment]
To generalize, rephrase to "AquaCrop v7.0 and higher versions" or remove the version 7.0 number.

**Comment by lbusschaert** [comment]
> To generalize, rephrase to "AquaCrop v7.0 and higher versions" or remove the version 7.0 number.

This was addressed. I will merge the changes.

**Pull request merged**